src/jit/compiler.h

```diff
@@ -598,6 +598,20 @@ class  LclVarDsc
         return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
     }
 
+    bool                lvIsMultiregStruct()
+    {
+#if FEATURE_MULTIREG_ARGS_OR_RET
+#ifdef _TARGET_ARM64_
+        if ((TypeGet() == TYP_STRUCT) &&
+            (lvSize()  == 2 * TARGET_POINTER_SIZE))
+        {
+            return true;
+        }
+#endif  // _TARGET_ARM64_
+#endif  // FEATURE_MULTIREG_ARGS_OR_RET
```

Comment:
I was thinking of the sizeof(void*) on an x86 breaking this while cross compiling to 64 bits.  Is it common practice to fix this on the fly or leave a comment tag?